tests/Fixer/Import/OrderedImportsFixerTest.php

```diff
@@ -780,7 +780,7 @@ public function provide70Cases()
     |--------------------------------------------------------------------------
     | Test sorting by length
     |--------------------------------------------------------------------------
-    */
+     */
```

Comment:
> `*` is an accepted standard, enforced in code and conventions by DocBlocks

I would say that it only applies to DocBlocks, which start with `/**`. Maybe the fixer should be less strict with regular multiline `/*` comments? or maybe not touch them at all?